lib/transforms/subsetFonts.js

```diff
@@ -218,8 +275,8 @@ async function getSubsetsForFontUsage(assetGraph, htmlAssetTextsWithProps, forma
                     subsetPromiseMap[promiseId].then(result => {
                         if (result) {
                             if (result.missingChars.length > 0) {
-                                missingCharsByFontSrc[fontUsage.props.src] =
-                                    result.missingChars.concat(missingCharsByFontSrc[fontUsage.props.src] || []);
+                                missingCharsByFontSrc[fontUsage.fontUrl] =
+                                    result.missingChars.concat(missingCharsByFontSrc[fontUsage.fontUrl] || []);
```

Comment:
Maybe it'd be more readable to use array spread:

```js
missingCharsByFontSrc[fontUsage.fontUrl] = [
    ...result.missingChars,
    ...(missingCharsByFontSrc[fontUsage.fontUrl] || [])
];
```

Reply:
Done